src/plugins/components/typography/wysiwyg.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Sets the vertical rhythm, i.e., spacing between elements such as heading, paragraphs and lists
+ * within unstructured WYSIWYG content.
+ */
+module.exports = function () {
+  return function ({ addComponents, theme }) {
+
+    const components = {
+      '.wysiwyg': {
+        '> *:first-child': {
+          marginTop: '-0.25em', // Leading trim
+        },
+        '> *:last-child': {
+          marginBottom: '0',
+        },
+        'p, ol, ul, dl': {
+          color: theme('color.black.true'),
+          marginBottom: '1.6em',
+        },
+        'li p': {
+          marginBottom: '0.4em',
+        },
+        'h2, h3, h4, h5, h6': {
+          marginBottom: '0.7em',
+          clear: 'both',
+          '&:not(:first-child)': {
+            marginTop: '1.6em',
```

Comment:
Perhaps I should have called out this one. Just want to call it out to make sure that having margin tops in wysiwyg is expected.

Reply:
Yup this has been tested. When there's a heading, I want to give it more space above so there's a larger separation between the heading and its previous element, and this looks better if it's proportional to the heading size (thus using em also). This is also due to the limitation of CSS - there's no way to select a previous element to give it a margin bottom (plus then it won't be proportional to the heading size).